docs/troubleshooting.md

```diff
@@ -80,3 +80,13 @@ end
 If instead you want to tune the Elasticsearch disk allocator's settings, please
 refer to
 [Disk-based shard allocation](https://www.elastic.co/guide/en/elasticsearch/reference/current/disk-allocator.html#disk-allocator).
+
+## CORS
+
+If you are experiencing CORS issues locally or are in need to display more
```

Comment:
`If you are experiencing CORS issues locally or are in need to display more` -> `If you are experiencing CORS issues locally or need to display more`